### ISSUE TITLE
Access levels: filters show categories for documents counting zero

### ISSUE THREAD

**Issue by txau** [huridocs/uwazi#3730]
![image](https://user-images.githubusercontent.com/489831/120954746-17a31c80-c70d-11eb-8a22-e7607e05b3c6.png)

In the screenshot, the group "Judges and Commissioners" has all of its templates not shared publicly, and yet the category appears. This adds noise to the UI and can be considered an information leak if the template is meant to be private.

**Comment by fnocetti** [comment]
Agree that this adds noise, but we don't have a concept of privacy on templates (anything different from entities actually) so this should not be considered a data leak.

**Comment by txau** [comment]
@fnocetti I agree with you in the sense of not being a critical information leak, but definitely an undesirable side effect. IMO this issue is low priority, but still worth fix at some point.
